src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,63 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Lotto {
+
+    private final List<LottoNumber> numberList;
+    private Rank rank;
+
+    public Lotto(final List<LottoNumber> numberList) {
+        this.numberList = numberList;
+    }
+
+    public List<LottoNumber> getNumberList() {
+        return numberList;
+    }
+
+    public Rank getRank() {
+        return rank;
+    }
+
+    public Lotto matchRank(List<Integer> winnerNumberList) {
+
+        List<LottoNumber> winnerNumberListToCompare = winnerNumberList.stream()
```

Comment:
```suggestion
    public Lotto matchRank(List<Integer> winnerNumberList) {
        List<LottoNumber> winnerNumberListToCompare = winnerNumberList.stream()
```
개행은 의미전달을 위해 많이 사용하는데 그래서 의미없는 개행은 최소화 하는게 좋을듯 해요!

Reply:
주의하도록 하겠습니다.